packages/vscode-docker-registries/src/clients/GitHub/GitHubRegistryDataProvider.ts

```diff
@@ -89,7 +89,8 @@ export class GitHubRegistryDataProvider extends RegistryV2DataProvider {
                     n: '100',
                     last: nextSearchString
                 },
-                scopes: ['registry:catalog:*']
+                scopes: ['registry:catalog:*'],
+                throwOnFailure: true,

```

Comment:
should we get rid of `throwOnFailure` and just always throw? since we have the error item now, it'll display nicely under the tree item

Reply:
Gotcha. I'll do some testing and if possible, make the change in the next PR.